compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -393,6 +393,11 @@ void addMultiTilingExpertPassPipeline(OpPassManager &passManager,
     }
   }
 
+  if (pipelineOpt.enableAArch64SSVE) {
```

Comment:
I missed it in the previous review. Should this happen after peeling? i.e., should it happen right before vectorization?

Reply:
Good point :+1: